### ISSUE TITLE
Uncaught Error: Cannot find module './docs\_template.jsx'

### ISSUE THREAD

**Issue by qinyadong** [gatsbyjs/gatsby#105]

**Comment by qinyadong** [comment]
in chrome it presents

**Comment by qinyadong** [comment]
how to handle this problem?

**Comment by KyleAMathews** [comment]
Looks like you need to change your backslash to a forward slash.

**Comment by cable729** [comment]
@KyleAMathews This is happening to me as well with a fresh install.

```
gatsby new blog https://github.com/gatsbyjs/gatsby-starter-blog
cd blog
gatsby develop
```

Window 10, Node 5.5.0, Gatsby 0.7.7.

**Comment by KyleAMathews** [comment]
Hi, can you both try the latest release 0.8?

**Comment by NogsMPLS** [comment]
just pulled latest and followed documentation start example instructions from README, get the same issue.

Windows 7, Node 5.6.0, npm 3.6.0.

I'll dig around and see if I can figure it out and hopefully submit a PR if I find anything.

**Comment by KyleAMathews** [comment]
That'd be great. I don't use Windows so weird stuff like this are beyond me.

**Comment by NogsMPLS** [comment]
messed around with it on my flight. i figured out a fix. its mainly got to do with how parsed-file is outputting dirname. just landed and got reactconf tomorrow, ill see if i can get a pr in for you to check on a mac sometime this week.

**Comment by KyleAMathews** [comment]
Oh cool! I'm going to be at react conf as well so perhaps we can have a
little gatsby powwow :-)

**Comment by KyleAMathews** [comment]
Published 0.8.4 with this.
